internal/auth/oidc.go

```diff
@@ -180,25 +180,25 @@ func (h *OIDCHandler) HandleLogin(w http.ResponseWriter, r *http.Request) {
 	http.Redirect(w, r, authURL, http.StatusFound)
 }
 
-func (h *OIDCHandler) HandleCallback(w http.ResponseWriter, r *http.Request) (string, error) {
+func (h *OIDCHandler) HandleCallback(w http.ResponseWriter, r *http.Request) (string, string, error) {
```

Comment:
Returning the whole claims struct as a pointer might be better here because then the caller can decide to grab username and extra variables.